index.bs

```diff
@@ -1233,7 +1235,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. [=set/Append=] |authenticator| to |issuedRequests|.
 
-1. While |issuedRequests| [=list/is not empty=], perform the following actions depending upon |lifetimeTimer|
+1. [=While=] |lifetimeTimer| has not expired, perform the following actions depending upon |lifetimeTimer|
```

Comment:
Why was the clause "issuedRequests not empty" deleted?

Reply:
See https://github.com/w3c/webauthn/pull/687#discussion_r161974515

Reply:
See https://github.com/w3c/webauthn/pull/687#issuecomment-358223418